typescript/src/chain.ts

```diff
@@ -18,6 +18,24 @@ export interface Identifier {
   identifierHex: string
 }
 
+/**
+ * Represents a generic chain event.
+ */
+export interface Event {
+  /**
+   * Ethereum block number of the event emission.
```

Comment:
This is a generic chain event placed in a generic `chain.ts` file. I think we should remove all mentions about "Ethereum":
```suggestion
   * Block number of the event emission.
```

Reply:
f816ce56c496c34b79d7fab46bf555c1688dc046